### DISCUSSION TITLE
PlexCleaner.json "ReEncodeVideo" everything but option?

### DISCUSSION THREAD

**Discussion by kmax9981** [ptr727/PlexCleaner#425] [Q&A] [answered]
I know that you can list formats and profiles in the PlexCleaner.json file that you want to ReEncode, however rather than listing a ton of formats, is there a way to say ReEncode everything but X instead?

For example, my convert options are below, I would like to have it convert anything that is not the below configuration

"ConvertOptions": {
    "FfMpegOptions": {
      "Video": "libx264 -crf 22 -preset medium -vf format=yuv420p",
      "Audio": "ac3",
      "Global": ""
    },
    "HandBrakeOptions": {
      "Video": "x264 --quality 22 --encoder-preset medium",
      "Audio": "copy --audio-fallback ac3"
    }

**Comment by ptr727** [comment, marked as answer]
No such ability The reason for matching is to list specifics that are problematic for directplay, not media normalization to a single type.

**Reply by kmax9981** [reply]
Thank you for the explanation
